src/views/dialogs/PreferencesDialog.tsx

```diff
@@ -13,6 +13,7 @@ export const PreferencesDialog = ({ setIsOpen }: DialogProps<PreferencesDialogPr
   return (
     <ComboboxDialogMenu
       isOpen
+      withSearch={false}
```

Comment:
unrelated nit but the content and title are not aligned :(((

<img width="552" alt="Screenshot 2024-10-18 at 11 47 45 AM" src="https://github.com/user-attachments/assets/70dd4724-00d5-4172-8b88-e7bc52120ea7">